Assets/PatchKit Patcher/Scripts/AppData/Local/Pack1Unarchiver.cs

```diff
@@ -23,18 +23,19 @@ public class Pack1Unarchiver : IUnarchiver
         private readonly string _packagePath;
         private readonly Pack1Meta _metaData;
         private readonly string _destinationDirPath;
+        private readonly string _suffix;
         private readonly byte[] _key;
         private readonly byte[] _iv;
 
         public event UnarchiveProgressChangedHandler UnarchiveProgressChanged;
 
-        public Pack1Unarchiver(string packagePath, Pack1Meta metaData, string destinationDirPath, string key)
-            : this(packagePath, metaData, destinationDirPath, Encoding.ASCII.GetBytes(key))
+        public Pack1Unarchiver(string packagePath, Pack1Meta metaData, string destinationDirPath, string key, string suffix = "")
```

Comment:
Instead of `""` you can use `string.Empty` - IMHO it is much more readable.

Reply:
Never mind - turns out `string.Empty` is not a compile time constant and it can't be used as default value for argument.